client/src/components/v3/status/WMStatus.tsx

```diff
@@ -22,6 +22,91 @@ export default function WMStatus(props: WMStatusProps) {
     </span>
   );
 
+  function convertToSentenceCase(str: String) {
+    /**
+     * convert Camel Case words to sentence case
+     *
+     * @param str - camel case string
+     * @returns string in sentence format
+     */
```

Comment:
Great documentation! Can you move the documentation to the line above the function definition? Different to what's traditional in python, but in most languages including JS/TS it's standard to put documentation above.